Serializer/Serializeables/NSValue+Serializable.m

```
@@ -0,0 +1,19 @@
+#import "NSValue+Serializable.h"
+
+@implementation NSValue(Serializable)
+
+- (NSString *)serializeWithError:(NSError **)error
+{
+	if (strcmp([self objCType], @encode(CGRect)) == 0)
+	{
+		CGRect rect = [self CGRectValue];
+
+		return [NSString stringWithFormat:@"CGRect: [(%f, %f), (%f, %f)]", rect.origin.x, rect.origin.y, rect.size.width, rect.size.height];
+	}
+	else
+	{
+		return nil;
```

Why not return an error here?